src/test/java/org/jenkinsci/plugins/codescene/Domain/DeltaAnalysisRequestTest.java

```diff
@@ -1,35 +1,129 @@
 package org.jenkinsci.plugins.codescene.Domain;
 
+import org.apache.commons.jelly.tags.fmt.Config;
+import org.apache.mina.core.RuntimeIoException;
 import org.junit.Test;
 
+import java.net.MalformedURLException;
+import java.net.URL;
+
 import static org.junit.Assert.*;
 
 public class DeltaAnalysisRequestTest {
 
     private static final Repository GIT_REPO = new Repository("codescene-ui");
     private static final int COUPLING_THRESHOLD = 65;
 
+    // emulate named parameters to get more calling context
+    private String useBiomarkers = "";
+    private boolean enableUserBiomarkers = false;
+    private static boolean enableFailedGoalGate = false;
+    private static boolean enableCodeHealthGate = false;
```

Comment:
I don't like this workaround for "named parameters".
The fields should be removed and userConfigFrom's params values just passed without assigning them to these fields.
If you really want to do it, then Builder pattern may be a better solution.

Reply:
Named params removed.